package.json

```diff
@@ -65,7 +65,7 @@
     "karma-jasmine": "~4.0.0",
     "karma-jasmine-html-reporter": "^1.5.0",
     "ng-packagr": "^12.1.2",
-    "node-sass": "^4.13.1",
+    "node-sass": "^7.0.0",
```

Comment:
can we get rid of node-sass completely and use dart-sass, drop this completely as `sass` is already a dependency of the angular build tools and instead change the scripts in `package.json` from `node-sass` to `sass`?

Reply:
sure, I was trying to make a minimum update to be able to build with node 16 and npm 8. I think when changing sass etc. we could also update the overall project including eslint etc.